barman/infofile.py

```diff
@@ -1020,7 +1021,7 @@ def is_checksum_consistent(self):
 
     def is_full_and_eligible_for_incremental(self):
         """
-        Used to filter out backups that have a parent backup id and are not
+        Used to filter out backups that are incremental and are not
         considered as FULL backups.
```

Comment:
```suggestion
        Check if this is a full backup taken with `postgres` method and which is
        eligible to be a parent for an incremental backup.
```

I wonder if this makes things clearer?
I looked a few times at the current summary, and it sounds confusing.